### PR TITLE
Implemented: Added entity and related data to store details for ups gateway connection details(#1te5zaw)

### PR DESCRIPTION
**Pull request by sheetalj2205**
Created entity UpsGateway with the package name "ups" for the connection with following fields:
*  upsGatewayConfigId
*  accessLicenceNumber
*  userId
*  password
* connected-url
* shipperNumber

Added demo data also of respective fields.

### Related Issues:
Ticket Link: [https://app.clickup.com/t/1te5zaw]

### CODE REVIEW

File: data/UpsDemoData.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<entity-facade-xml type="demo">
+    <ups.ShippingGatewayUps shippingGatewayConfigId="UpsDemo" accessLicenceNumber="" userId="" password="" shipperNumber="" connectUrl="https://onlinetools.ups.com/rest/"/>
```
Comment (dixitdeepak) [inline data/UpsDemoData.xml]: Does `https://onlinetools.ups.com/rest` is for sandbox account ?

**Comment by sheetalj2205** [comment]
@dixitdeepak  Please review this PR.

File: entity/UpsEntities.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<entities xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/entity-definition-2.1.xsd">
+    <entity entity-name="ShippingGatewayUps" package="ups">
```
Comment (dixitdeepak) [inline entity/UpsEntities.xml:3]: Do we have some more configuration setting that should be part of the entity definition?

File: data/UpsDemoData.xml (hunk 1 above)
Comment (sheetalj2205) [reply data/UpsDemoData.xml]: No! Sir. This is the production URL as this is the only url which is same for all services.

File: entity/UpsEntities.xml (hunk 2 above)
Comment (sheetalj2205) [reply entity/UpsEntities.xml:3]: No! they are sufficient.

**Comment by sheetalj2205** [comment]
@dixitdeepak Please check now

File: entity/UpsEntities.xml (hunk 2 above)
Comment (dixitdeepak) [reply entity/UpsEntities.xml:3]: ok

**Review by dixitdeepak — APPROVED** [review]

**Pull request merged**